Project/Project/Views/Auth/Login.cshtml

```diff
@@ -0,0 +1,35 @@
+﻿@model LoginViewModel
+
+@{
+    ViewData["Title"] = "Login";
+}
+
+<section class="">
+    <div class="row d-flex justify-content-center align-items-center p-5">
+        <div class="col-6 col-md-3">
+            <form asp-action="login" method="post" class="rounded p-5 bg-secondary-color ">
+                <div class="d-flex flex-column justify-content-center align-items-center">
+                    <h2 class="text-center fw-bold mb-4 display-6">Sign In</h2>
+
+                    <div asp-validation-summary="ModelOnly" class="text-danger" role="alert"></div>
+
+                    <label asp-for="Email" class="col-md-4 col-form-label align-self-start">Email</label>
+                    <input asp-for="Email" class="form-control w-100 rounded-pill mb-2" type="email" name="email" id="email" autocomplete="email" aria-required="true" placeholder="Username" />
```

Comment:
Decide if Username or Email, dont ask for Email and give Username as example value